Parking_Lot_Reservation/Models/PersonModel.cs

```diff
@@ -13,8 +13,6 @@ public class PersonModel
         public int PersonId { get; set; }
```

Comment:
No need of adding [Key] attribute. If Name of int variable contains "Id" EntityFramework will come up itself on that.

Reply:
I forgot about that. Thanks :)